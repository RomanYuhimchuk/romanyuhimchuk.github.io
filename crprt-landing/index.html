<!DOCTYPE html>
<html lang="en">
<head>
 <meta charset="UTF-8">
 <title>FOLIO</title>
 <!-- CSS -->
<link href='https://fonts.googleapis.com/css?family=Exo:400,100,500,400italic' rel='stylesheet' type='text/css'>
<link href='https://fonts.googleapis.com/css?family=Lato' rel='stylesheet' type='text/css'>
 <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" integrity="sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7" crossorigin="anonymous">
 <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css">
 <link href="stylesheets/screen.css" media="screen, projection" rel="stylesheet" type="text/css">
 <!-- JS -->
 <script src="https://code.jquery.com/jquery-2.2.0.min.js"></script>
 <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js" integrity="sha384-0mSbJDEHialfmuBBQP6A4Qrprq5OVfW37PRR3j5ELqxss1yVqOtnepnHVP9aJ7xS" crossorigin="anonymous"></script>
 </head>
<body>
<header>
	<h1>There is no other platforms for you as like </h1>
	<a href="#"><i class="fa fa-play fa-lg"></i></a>
	<button class="btn cl-btn">try now</button>
	<p>* No need to add cards details</p>
</header>
<section>
	<div class="cl-design clearfix">
		<div class="cl-design-right pull-right"></div>
		<div class="cl-design-left">
			<h4>new design</h4>
			<h2>There is no other platforms for you as like <span>....</span></h2>
			<p>Aenean eu leo quam. Pellentesque ornare sem lacinia quam venenatis vestibulum. Donec id elit non mi porta gravida at eget metus. Curabitur blandit tempus porttitor. Etiam porta sem malesuada magna mollis euismod. Vestibulum id ligula porta felis euismod semper.</p>
			<p>Donec sed odio dui. Aenean lacinia bibendum nulla sed consectetur. Donec ullamcorper nulla non metus auctor fringilla. Aenean lacinia bibendum nulla sed consectetur. Nulla vitae elit libero, a pharetra augue.</p>
		</div>
	</div>
	<div class="cl-features">
		<h4>new features</h4>
		<h2>Some awesome features</h2>
		<div class="container">
			<div class="row">
				<div class="col-md-4 col-sm-4">
					<img src="images/icon_01.png" alt="logo1">
					<h3>Some awesone features</h3>
					<p>Donec sed odio dui. Aenean lacinia bibendum nulla sed consectetur. Nulla vitae elit libero, a pharetra augue.</p>
				</div>
				<div class="col-md-4 col-sm-4">
					<img src="images/icon_02.png" alt="logo1">
					<h3>Some awesone features</h3>
					<p>Donec sed odio dui. Aenean lacinia bibendum nulla sed consectetur. Nulla vitae elit libero, a pharetra augue.</p>
				</div>
				<div class="col-md-4 col-sm-4">
					<img src="images/icon_03.png" alt="logo1">
					<h3>Some awesone features</h3>
					<p>Donec sed odio dui. Aenean lacinia bibendum nulla sed consectetur. Nulla vitae elit libero, a pharetra augue.</p>
				</div>
			</div>
		</div>
	</div>
	<div class="cl-responsive clearfix">
		<img src="images/comp.png" alt="computer image" class="img-responsive">
		<div class="cl-responsive-right">
			<h4>new design</h4>
			<h2>Resposive design, just need your tap <span>....</span></h2>
			<p>Aenean eu leo quam. Pellentesque ornare sem lacinia quam venenatis vestibulum. Donec id elit non mi porta gravida at eget metus. Curabitur blandit tempus porttitor. Etiam porta sem malesuada magna mollis euismod. Vestibulum id ligula porta felis euismod semper.</p>
		</div>		
	</div>
	<div class="cl-founders">
		<div class="container">
			<div class="row">
				<div class="col-md-4 col-sm-4">
					<img src="images/user.jpg" alt="founder photo">
					<h3>Bob Marley</h3>
					<h4>co-founder</h4>
					<p>“Aenean eu leo quam. Pellentesque ornare sem lacinia quam venenatis vestibulum. Donec id gravida at eget metus. Etiasem malesuada magn”</p>
				</div>
				<div class="col-md-4 col-sm-4">
					<img src="images/user.jpg" alt="founder photo">
					<h3>Bob Marley</h3>
					<h4>co-founder</h4>
					<p>“Aenean eu leo quam. Pellentesque ornare sem lacinia quam venenatis vestibulum. Donec id gravida at eget metus. Etiasem malesuada magn”</p>
				</div>
				<div class="col-md-4 col-sm-4">
					<img src="images/user.jpg" alt="founder photo">
					<h3>Bob Marley</h3>
					<h4>co-founder</h4>
					<p>“Aenean eu leo quam. Pellentesque ornare sem lacinia quam venenatis vestibulum. Donec id gravida at eget metus. Etiasem malesuada magn”</p>
				</div>
			</div>
		</div>
		<div class="cl-founders-logos">
		<img src="images/logos_01.png" alt="client logo">
		<img src="images/logos_02.png" alt="client logo">
		<img src="images/logos_03.png" alt="client logo">
		<img src="images/logos_04.png" alt="client logo">
		<img src="images/logos_05.png" alt="client logo">
		</div>
	</div>
	<div class="cl-center">
		<h4>new features</h4>
		<h2>Some awesome features</h2>
		<p>Aenean eu leo quam. Pellentesque ornare sem lacinia quam venenatis vestibulum. Donec id elit non mi porta gravida at eget metus. Curabitur blandit tempus porttitor. Etiasem malesuada magn. Vestibulum felis euismod semper.</p>
		<img src="images/back_11.png" alt="page image" class="img-responsive">
	</div>
	<div class="cl-form">
		<h4>new features</h4>
		<h2>Over 1000 designers are using ...</h2>
		<div class="container">
		<form method="post" action="#">
		<div class="row">
			<div class="col-md-3">
				<input type="text" name="name" placeholder="full name"/>
			</div>
			<div class="col-md-3">
				<input type="email" name="email" placeholder="your email"/>
			</div>
			<div class="col-md-3">
				<input type="password" name="pass" placeholder="password"/>
			</div>
			<div class="col-md-3">
				<span class="pull-left"><img src="images/arrow.png">30 days free trial for all</span>
				<button type="submit" class="btn cl-btn">try now</button>
			</div>
		</div>
		</form>
		</div>
		<p>By Signing up you agree to our <a href="#">terms & Services</a>.</p>
	</div>
</section>
<footer>
	<div class="container">
		<div class="row">
			<div class="col-md-3 col-sm-12">
				<div class="cl-footer-logo">
					Start<i class="fa fa-play fa-4x"><span>Ex</span></i>
				</div>
				<p>Aenean lacinia bibendum nulla sed consectetur. Aenean eu leo quam. USA &  CAN: 1-888-123-4567 Address: 34 Brokel Rd. NY</p>
			</div>
			<div class="col-md-2 col-sm-6">
				<h3>support</h3>
				<a href="#">help center</a>
				<a href="#">get started</a>
				<a href="#">contact us</a>
			</div>
			<div class="col-md-2 col-sm-6">
				<h3>about us</h3>
				<a href="#">about us</a>
				<a href="#">terms of use</a>
				<a href="#">privacy policy</a>
			</div>
			<div class="col-md-3 col-sm-12">
				<h3>get newsletter</h3>
				<form>
					<input type="email" placeholder="EMAIL" name="email-footer"/>
					<button type="submit" class="btn cl-btn"><i class="fa fa-paper-plane-o fa-lg"></i></button>
				</form>
				<div class="cl-footer-logos">
					<a href="#"><i class="fa fa-dribbble fa-lg"></i></a>
					<a href="#"><i class="fa fa-facebook fa-lg"></i></a>
					<a href="#"><i class="fa fa-google-plus fa-lg"></i></a>
					<a href="#"><i class="fa fa-twitter fa-lg"></i></a>
				</div>
			</div>
		</div>
	</div>
</footer>
</body>
</html>