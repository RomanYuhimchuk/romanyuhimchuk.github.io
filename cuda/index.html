<!doctype html>
<html>
<head>
<meta charset="utf-8">
<title>Flat user interface</title>
<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" integrity="sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7" crossorigin="anonymous">
<link href='https://fonts.googleapis.com/css?family=Titillium+Web:400,600,700,300' rel='stylesheet' type='text/css'>
<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css">
<link rel="stylesheet" href="style.css">
<script type="text/javascript" src="https://code.jquery.com/jquery-2.1.4.min.js"></script>
<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js" integrity="sha384-0mSbJDEHialfmuBBQP6A4Qrprq5OVfW37PRR3j5ELqxss1yVqOtnepnHVP9aJ7xS" crossorigin="anonymous"></script>
</head>
<body>
<header>
  <div class="container">
  <div class="ex-menu clearfix">
     <a href="#" class="pull-left"><img src="images/logo1.png" alt="small logo">
     </a>
		<div class="pull-right">
  	      <ul>
			<li class="active"><a href="#">home</a></li>
			<li><a href="#">about</a></li>
			<li><a href="#">work</a></li>
			<li><a href="#">blog</a></li>
			<li><a href="#">contact</a></li>
          </ul>
          <div class="ex-hidden-menu">
          		<label for="ex-menu"><i class="fa fa-bars fa-2x"></i></label>
          		<input type="checkbox" id="ex-menu" checked>
          		<ul>
					<li><a href="#">home</a></li>
					<li><a href="#">about</a></li>
					<li><a href="#">work</a></li>
					<li><a href="#">blog</a></li>
					<li><a href="#">contact</a></li>
          		</ul>
          </div>
  		</div>
  </div>
  	<h1>Hi there! We are the new kids on the block 
	and we build awesome websites and mobile apps.</h1>
	<button type="button" class="btn btn-success ex-btn">WORK WITH US!<div></div></button>
	</div>
</header>
<section>
	<div class="ex-logo">
	<div class="container">
		<h1>SERVICES WE PROVIDE</h1>
		<div class="ex-underline">&nbsp;</div>	
		<h4>We are working with both individuals and businesses from all over the globe <br/>to create awesome websites and applications.
		</h4>
		<div class="row ex-marg">	
		<div class="col-md-3 col-sm-6 col-xs-12">
			<div><img src="images/logo2.png" alt="logo"></div>
			<h3>Branding</h3>
			<p>Lorem ipsum dolor sit amet, 
consectetuer adipiscing elit, sed diam nonummy nibh.</p>
		</div>
		<div class="col-md-3 col-sm-6 col-xs-12">
			<div><img src="images/logo3.png" alt="logo"></div>
			<h3>design</h3>
			<p>At vero eos et accusamus et iusto odio dignissimos qui blanditiis praesentium. </p>
		</div>
		<div class="col-md-3 col-sm-6 col-xs-12">
			<div><img src="images/logo4.png" alt="logo"></div>
			<h3>development</h3>
			<p>At vero eos et accusamus et iusto odio dignissimos qui blanditiis praesentium.
			</p>
		</div>
		<div class="col-md-3 col-sm-6 col-xs-12">
			<div><img src="images/logo5.png" alt="logo"></div>
			<h3>rocket science</h3>
			<p>Et harum quidem rerum est et expedita distinctio. Nam libero tempore.</p>
		</div>
		</div>
	</div>
	</div>
	<div class="ex-userpic">
	<div class="container">
		<h1>MEET OUR BEAUTIFUL TEAM</h1>
		<div class="ex-underline">&nbsp;</div>	
		<h4>We are a small team of designers and developers, who help brands with big ideas.
		</h4>
		<div class="row ex-marg">
		<div class="col-md-3 col-sm-6 col-xs-12">
			<img src="images/userpic1.jpg" alt="logo">
			<h3>jack white</h3>
			<h4>the white stripes</h4>
			<p>Lorem ipsum dolor sit amet, 
			consectetuer adipiscing elit, sed diam nonummy nibh euismod tincidunt ut laoreet dolore magna.</p>
			<a href="#"><i class="fa fa-facebook"></i></a>
			<a href="#"><i class="fa fa-twitter"></i></a>
			<a href="#"><i class="fa fa-linkedin"></i></a>
			<a href="#"><i class="fa fa-envelope"></i></a>
		</div>
		<div class="col-md-3 col-sm-6 col-xs-12">
			<img src="images/userpic2.jpg" alt="logo">
			<h3>bob marley</h3>
			<h4>bob marley & the wailers</h4>
			<p>Duis aute irure dolor in in voluptate velit esse cillum dolore fugiat nulla pariatur. Excepteur sint occaecat non diam proident. </p>
			<a href="#"><i class="fa fa-twitter"></i></a>
			<a href="#"><i class="fa fa-linkedin"></i></a>
			<a href="#"><i class="fa fa-envelope"></i></a>
		</div>
		<div class="col-md-3 col-sm-6 col-xs-12">
			<img src="images/userpic3.jpg" alt="logo">
			<h3>mike patton</h3>
			<h4>faith no more</h4>
			<p>Nemo enim ipsam voluptas sit aspernatur aut odit aut fugit, sed quia consequuntur magni dolores eos qui ratione voluptatem nesciunt.
			</p>
			<a href="#"><i class="fa fa-facebook"></i></a>
			<a href="#"><i class="fa fa-twitter"></i></a>
			<a href="#"><i class="fa fa-linkedin"></i></a>
			<a href="#"><i class="fa fa-envelope"></i></a>
		</div>
		<div class="col-md-3 col-sm-6 col-xs-12">
			<img src="images/userpic4.jpg" alt="logo">
			<h3>nick cave</h3>
			<h4>nick cave & the bad seeds</h4>
			<p>Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam.</p>
			<a href="#"><i class="fa fa-facebook"></i></a>
			<a href="#"><i class="fa fa-twitter"></i></a>
			<a href="#"><i class="fa fa-envelope"></i></a>
		</div>
		</div>
	</div>
	</div>
	<div class="ex-form">
	<div class="container">
		<h1>get in touch</h1>
		<div class="ex-underline">&nbsp;</div>	
		<h4>1600 Pennsylvania Ave NW, Washington, DC 20500, United States of America. Tel: (202) 456-1111
		</h4>
		<form method="post">
			<input type="text" class="ex-input" placeholder="Your Name">
			<input type="text" class="ex-input" placeholder="Your Email">
			<textarea col="5" placeholder="Your Message" class="ex-input"></textarea>
			<button type="submit" class="btn btn-success ex-btn">send message<div></div></button>
		</form>
	</div>
	</div>
</section>
<footer>
	<div class="container">	
	<ul>
		<li><a href="#">facebook</a></li>
		<li><a href="#" class="active">twitter</a></li>
		<li><a href="#">google+</a></li>
		<li><a href="#">linkedin</a></li>
		<li><a href="#">behance</a></li>
		<li><a href="#">dribble</a></li>
		<li><a href="#">gitHub</a></li>
	</ul>
	<ul>
		<li><a href="#"><i class="fa fa-facebook-square fa-2x"></i></a></li>
		<li><a href="#" class="active"><i class="fa fa-twitter-square fa-2x"></i></a></li>
		<li><a href="#"><i class="fa fa-google-plus-square fa-2x"></i></a></li>
		<li><a href="#"><i class="fa fa-linkedin-square fa-2x"></i></a></li>
		<li><a href="#"><i class="fa fa-behance-square fa-2x"></i></a></li>
		<li><a href="#"><i class="fa fa-dribbble fa-2x"></i></a></li>
		<li><a href="#"><i class="fa fa-github-square fa-2x"></i></a></li>
	</ul>
	</div>
</footer>
</body>
</html>