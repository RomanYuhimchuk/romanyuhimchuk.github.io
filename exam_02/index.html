<!DOCTYPE html>
<html>
<head lang="en">
	<title>Elegant</title>
	<meta http-equiv="X-UA-Compatible" content="IE=edge">
	<meta name="viewport" content="width=device-width, initial-scale=1">
	<meta charset="UTF-8">
	<meta name="description" content="" />
	<meta name="keywords" content="" />
	<link rel="shortcut icon" href="favicon.png"/>
	
	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" integrity="sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7" crossorigin="anonymous">
	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css">
	<link href="exam.css" rel="stylesheet" type="text/css">
	<script src="https://code.jquery.com/jquery-2.2.0.min.js"></script>
	<script> 
$(document).ready(function(){
    $(".elgnt-menu > i").click(function(){
        $(".elgnt-menu ul").css({
        	"position":"absolute",
        	"top":"80px",
        	"right":"15px"
        	});
        $(".elgnt-menu ul li").css({
        	"display":"block",
        	"text-align":"right"
        	});
        $(".elgnt-menu ul").slideToggle();
    });
});
</script>
	<!--[if lt IE 9]>
		<script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
		<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
	<![endif]-->
</head>
<body>
	<header>
		<div class="container">
			<div class="elgnt-menu">
				<a href="#"><img src="images/logo.png" alt="logo"></a>
					<ul class="pull-right">
						<li><a href="#">hello</a></li>
						<li><a href="#services">services</a></li>
						<li><a href="#about">about us</a></li>
						<li><a href="#work">our work</a></li>
						<li><a href="#contact">contact us</a></li>
					</ul>
					<i class="fa fa-bars fa-2x"></i>
			</div>
			<h1 class="elgnt-heading elgnt-light">weâ€™re creative agency</h1>
			<p>Lorem ipsum dolor sit amet, consectetuer dipiscing elit. Aenean commodo ligula eget dolor. Aenean massa.</p>
			<a href="#services"><span><i class="fa fa-chevron-down fa-2x"></i></span></a>
		</div>
	</header>
	<section>
		<div class="elgnt-services" id="services">
			<div class="container">
				<h3 class="elgnt-heading elgnt-dark">our services</h3>
				<div class="row">
					<div class="col-md-4">
						<div>
							<img src="images/services_1.svg" alt="service img">
						</div>
						<h4>app design</h4>
						<p>Lorem ipsum dolor sit amet, onsectetur adipisicing elit, sed do eiusmod tempor ilabore et dolore.</p>
					</div>
					<div class="col-md-4">
						<div>
							<img src="images/services_2.svg" alt="service img">
						</div>
						<h4>responsive layout</h4>
						<p>Lorem ipsum dolor sit amet, onsectetur adipisicing elit, sed do eiusmod tempor ilabore et dolore.</p>
					</div>
					<div class="col-md-4">
						<div>
							<img src="images/services_3.svg" alt="service img">
						</div>
						<h4>pixel perfect design</h4>
						<p>Lorem ipsum dolor sit amet, onsectetur adipisicing elit, sed do eiusmod tempor ilabore et dolore.</p>
					</div>
				</div>
			</div>
		</div>
		<div class="elgnt-about" id="about">
			<div class="container">
				<div class="row">
					<div class="col-md-7 col-sm-7 col-xs-12">
						<h2 class="elgnt-light">dolore magna aliqua</h2>
						<p>Lorem ipsum dolor sit amet,consectetur adipisicing elit,sed do eiusmod tempor incididunt ut laboreet dolore magna aliqua.</p>
						<a href="#" class="elgnt-link">more</a>
					</div>
					<div class="col-md-3 col-md-offset-1 col-sm-offset-0 col-sm-5 col-xs-12">
					<img src="images/back2_1.png" alt="product photo">
					</div>
				</div>
			</div>
		</div>
		<div class="elgnt-work" id="work">
			<div class="container">
			<h3 class="elgnt-heading elgnt-dark">our work</h3>
			<div class="row">
				<div class="col-md-4 col-sm-4 col-xs-12">
				<div class="elgnt-work-img">
					<img src="images/work_1.jpg" alt="work img">
					<a href="#" class="elgnt-work-hidden">
						<span>
						<p>kinderen planet</p>
						<i class="fa fa-link fa-2x"></i>
						</span>
					</a>
				</div>
				</div>
				<div class="col-md-4 col-sm-4 col-xs-12">
				<div class="elgnt-work-img">
					<img src="images/work_2.jpg" alt="work img">
					<a href="#" class="elgnt-work-hidden">
						<span>
						<p>jetindus airlines</p>
						<i class="fa fa-link fa-2x"></i>
						</span>
					</a>
				</div>
				</div>
				<div class="col-md-4 col-sm-4 col-xs-12">
				<div class="elgnt-work-img">
					<img src="images/work_3.jpg" alt="work img">
					<a href="#" class="elgnt-work-hidden">
						<span>
						<p>trimplement</p>
						<i class="fa fa-link fa-2x"></i>
						</span>
					</a>
				</div>
				</div>
			</div>
			<a href="#" class="elgnt-btn">more</a>
			</div>
		</div>
	</section>
	<footer>
		<div class="elgnt-form" id="contact">
			<h3 class="elgnt-heading elgnt-light">get in touch</h3>
			<div class="elgnt-form-icons">
				<a href="#"><i class="fa fa-twitter fa-2x"></i></a>
				<a href="#"><i class="fa fa-facebook fa-2x"></i></a>
				<a href="#"><i class="fa fa-linkedin fa-2x"></i></a>
				<a href="#"><i class="fa fa-vimeo fa-2x"></i></a>
				<a href="#"><i class="fa fa-behance fa-2x"></i></a>
			</div>
			<form method="post" action="#">
				<input class="pull-left" type="text" placeholder="name">
				<input class="pull-right" type="email" placeholder="email">
				<textarea rows="1" placeholder="message"></textarea>
				<input type="submit" class="elgnt-btn" value="send">
			</form>
		</div>
	</footer>
</body>
</html>