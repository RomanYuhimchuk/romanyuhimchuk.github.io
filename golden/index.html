<!DOCTYPE html>
<html lang="en">
<head>
 <meta charset="UTF-8">
 <title>Golden</title>
 <!-- CSS -->
 <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" integrity="sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7" crossorigin="anonymous">
 <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css">
 <link rel="shortcut icon" href="favicon.ico" type="image/x-icon">
 <link href="style.css" rel="stylesheet" type="text/css">
 <!-- JS -->
 <script src="https://code.jquery.com/jquery-2.2.0.min.js"></script>
 <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js" integrity="sha384-0mSbJDEHialfmuBBQP6A4Qrprq5OVfW37PRR3j5ELqxss1yVqOtnepnHVP9aJ7xS" crossorigin="anonymous"></script>
 <script> 
$(document).ready(function(){
    $(".gldn-menu-top span").click(function(){
        $(".gldn-menu-top ul").css({
        	"position":"absolute",
        	"top":"30px",
        	"right":"10px"
        	});
        $(".gldn-menu-top ul li").css({
        	"display":"block",
        	"margin-bottom":"10px"
        	});
        $(".gldn-menu-top ul").slideToggle("slow");
    });
});
</script>
 </head>
<body>
<header>
<div class="container">
	<div class="gldn-menu">
		<div class="gldn-menu-top clearfix">
		<a href="#"><img src="images/logo.png" class="pull-left" alt="logo"></a>
		<ul class="pull-right">
			<li><a href="#">home</a></li>
			<li><a href="#services">services</a></li>
			<li><a href="#portfolio">portfolio</a></li>
			<li><a href="#about">about</a></li>
			<li><a href="#contact">contact</a></li>
		</ul>
		<span><i class="fa fa-bars fa-2x"></i></span>
		</div>
		<div>Welcome To Our Studio!</div>
		<h1>It’s nice to Meet you</h1>
		<a href="#" class="gldn-btn">tell me more</a>
	</div>
</div>
</header>
<section>
<div class="gldn-services" id="services">
	<div class="container">
	<h2>services</h2>
	<p>Proin iaculis purus consequat sem cure. </p>
	<div class="row">
		<div class="col-md-4 col-sm-4">
			<div class="gldn-services-round">
				<i class="fa fa-shopping-basket fa-3x"></i>
			</div>
			<h3>E-Commerce</h3>
			<p>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.</p>
		</div>
		<div class="col-md-4 col-sm-4">
			<div class="gldn-services-round">
				<i class="fa fa-laptop fa-3x"></i>
			</div>
			<h3>responsive web</h3>
			<p>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.</p>
		</div>
		<div class="col-md-4 col-sm-4">
			<div class="gldn-services-round">
				<i class="fa fa-lock fa-3x"></i>
			</div>
			<h3>web security</h3>
			<p>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.</p>
		</div>
	</div>
	</div>
</div>
<div class="gldn-portfolio" id="portfolio">
	<div class="container">
		<h2>our portfolio</h2>
		<p>Proin iaculis purus consequat sem cure.</p>
		<div class="row">
			<div class="col-md-4 col-sm-6">
				<img src="images/prtfl_1.jpg" alt="portfolio image">
				<h3>ebony &amp; ivory</h3>
				<p>branding</p>
			</div>
			<div class="col-md-4 col-sm-6">
				<img src="images/prtfl_2.jpg" alt="portfolio image">
				<h3>smart stationary</h3>
				<p>print design</p>
			</div>
			<div class="col-md-4 col-sm-6">
				<img src="images/prtfl_3.jpg" alt="portfolio image">
				<h3>clever poster</h3>
				<p>print design</p>
			</div>
			<div class="col-md-4 col-sm-6">
				<img src="images/prtfl_4.jpg" alt="portfolio image">
				<h3>vinyl record</h3>
				<p>Product Mock-Up</p>
			</div>
			<div class="col-md-4 col-sm-6">
				<img src="images/prtfl_5.jpg" alt="portfolio image">
				<h3>treehouse template</h3>
				<p>web design</p>
			</div>
			<div class="col-md-4 col-sm-6">
				<img src="images/prtfl_6.jpg" alt="portfolio image">
				<h3>burned logo</h3>
				<p>branding</p>
			</div>
		</div>
	</div>
</div>
<div class="gldn-about" id="about">
	<div class="container">
	<h2>about us</h2>
	<p>Proin iaculis purus consequat sem cure.</p>
		<div class="row">
			<div class="col-md-3 col-md-offset-1 col-sm-offset-1 col-sm-5 col-xs-12">
				<img src="images/back_1.jpg" class="gldn-about-hidden" alt="timeline photo">
				<h3>july 2010<br/>Our Humble Beginnings</h3>
				<p>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt. Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit.</p>
			</div>
			<div class="col-md-4 col-sm-6 col-xs-12">
				<div>
				<span><img src="images/back_1.jpg" alt="timeline photo"></span>
				</div>
			</div>
		</div>
		<div class="row">
			<div class="col-md-push-4 col-md-4 col-sm-push-6 col-sm-6 col-xs-12">
				<div>
				<span><img src="images/back_2.jpg" alt="timeline photo"></span>
				</div>
			</div>
			<div class="col-md-3 col-md-push-3 col-sm-5 col-sm-offset-1 col-sm-pull-6 col-xs-12">
				<img src="images/back_2.jpg" class="gldn-about-hidden" alt="timeline photo">
				<h3>january 2011<br/>Facing Startup Battles</h3>
				<p>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt. Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit.</p>
			</div>
		</div>
		<div class="row">
			<div class="col-md-3 col-md-offset-1 col-sm-offset-1 col-sm-5 col-xs-12">
				<img src="images/back_3.jpg" class="gldn-about-hidden" alt="timeline photo">
				<h3>december 2012<br/>Enter The Dark Days</h3>
				<p>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt. Proin iaculis purus consequat sem cure digni.</p>
			</div>
			<div class="col-md-4 col-sm-6 col-xs-12">
				<div>
				<span><img src="images/back_3.jpg" alt="timeline photo"></span>
				</div>
			</div>
		</div>
		<div class="row">
			<div class="col-md-push-4 col-md-4 col-sm-push-6 col-sm-6 col-xs-12">
				<div>
				<span><img src="images/back_4.jpg" alt="timeline photo"></span>
				</div>
			</div>
			<div class="col-md-3 col-md-push-3 col-sm-5 col-sm-offset-1 col-sm-pull-6 col-xs-12">
				<img src="images/back_4.jpg" class="gldn-about-hidden" alt="timeline photo">
				<h3>february 2014<br/>Our Triumph</h3>
				<p>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt. Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean.</p>
			</div>
		</div>
		<div class="row">
			<div class="col-md-4 col-md-offset-4 col-sm-6 col-sm-offset-6 col-xs-12">
				<span>our story continues ...</span>
			</div>
		</div>
	</div>
</div>
<div class="gldn-team">
	<div class="container">
		<h2>our amazing team</h2>
		<p>Proin iaculis purus consequat sem cure.</p>
		<div class="row">
			<div class="col-md-4 col-sm-4">
				<img src="images/userpic1.jpg" alt="userpic">
				<h4>Iggy Pop</h4>
				<p>marketer</p>
				<a href="#" class="gldn-rnd-social"><i class="fa fa-twitter fa-lg"></i></a>
				<a href="#" class="gldn-rnd-social"><i class="fa fa-facebook fa-lg"></i></a>
				<a href="#" class="gldn-rnd-social"><i class="fa fa-pinterest-p fa-lg"></i></a>
			</div>
			<div class="col-md-4 col-sm-4">
				<img src="images/userpic2.jpg" alt="userpic">
				<h4>David Bowie</h4>
				<p>coder</p>
				<a href="#" class="gldn-rnd-social"><i class="fa fa-twitter fa-lg"></i></a>
				<a href="#" class="gldn-rnd-social"><i class="fa fa-facebook fa-lg"></i></a>
				<a href="#" class="gldn-rnd-social"><i class="fa fa-google-plus fa-lg"></i></a>
			</div>
			<div class="col-md-4 col-sm-4">
				<img src="images/userpic3.jpg" alt="userpic">
				<h4>sid vicious</h4>
				<p>graphic designer</p>
				<a href="#" class="gldn-rnd-social"><i class="fa fa-facebook fa-lg"></i></a>
				<a href="#" class="gldn-rnd-social"><i class="fa fa-pinterest-p fa-lg"></i></a>
			</div>
		</div>
		<p>Proin iaculis purus consequat sem cure  digni ssim donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt proin iaculis.</p>
	</div>
</div>
<div class="gldn-partners">
	<img src="images/prtnr_1.png" alt="partner logo">
	<img src="images/prtnr_2.png" alt="partner logo">
	<img src="images/prtnr_3.png" alt="partner logo">
	<img src="images/prtnr_4.png" alt="partner logo">
</div>
<div class="gldn-contact" id="contact">
	<div class="container">
	<h2>contact us</h2>
	<p>Proin iaculis purus consequat sem cure.</p>
	<form method="get" action="#">
		<div class="row">
			<div class="col-md-6">
				<input type="text" name="name" placeholder="your name*">
				<input type="email" name="email" placeholder="your e-mail*">
				<input type="text" name="subject" placeholder="subject">
			</div>
			<div class="col-md-6">
				<textarea name="text" placeholder="your message*"></textarea>
			</div>
		</div>
		<button type="submit" class="gldn-btn">send message</button>
	</form>
	</div>
</div>
</section>
<footer>
<div class="container">
<a href="#" class="gldn-rnd-social"><i class="fa fa-twitter fa-lg"></i></a>
<a href="#" class="gldn-rnd-social"><i class="fa fa-facebook fa-lg"></i></a>
<a href="#" class="gldn-rnd-social"><i class="fa fa-pinterest-p fa-lg"></i></a>
<a href="#" class="gldn-rnd-social"><i class="fa fa-google-plus fa-lg"></i></a>
<span>© Copyright 2014 FreebiesXpress.com</span>
</div>
</footer>
</body>
</html>